contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -65,9 +63,11 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         return validators[validatorId].amount;
     }
 
-    function _topUpForFee(address user, uint256 amount) private {
-        totalHeimdallFee = totalHeimdallFee.add(amount);
-        logger.logTopUpFee(user, amount);
+    function _transferAndTopUp(address user, uint256 fee, uint256 additionalAmount) private {
+        require(fee >= minHeimdallFee, "Minimum amount is 1 Matic");
```

Comment:
can use "min amount is less than the threshold" or something instead of 1 Matic.

Reply:
yeah, will do